internal/config/config_helpers.go

```diff
@@ -24,5 +25,12 @@ func ParseUndecodedKeys(undecodedKeys []toml.Key) string {
 	for errorKey := range unusedKeys {
 		errorKeys = append(errorKeys, style.Symbol(errorKey))
 	}
-	return strings.Join(errorKeys, ", ")
+
+	pluralizedElement := "element"
```

Comment:
If we are going to return a string, we might as well also return the "element(s)" part as well, to remove more duplication.